apps/arweave/src/ar_data_sync.erl

```diff
@@ -389,6 +389,10 @@ is_chunk_cache_full() ->
 			not_initialized
 	end.
 
+-ifdef(DEBUG).
+is_disk_space_sufficient(_StoreID) ->
+	true.
+-else.
```

Comment:
we run some nodes in debug mode, I wonder if this will cause the disk to fill up?

Reply:
ok, bad naming, but makes sense then